src/main/scala/shade/memcached/internals/SpyMemcachedIntegration.scala

```diff
@@ -173,20 +173,11 @@ class SpyMemcachedIntegration(cf: ConnectionFactory, addrs: Seq[InetSocketAddres
 
     val op: GetOperation = opFact.get(key, new GetOperation.Callback {
       def receivedStatus(opStatus: OperationStatus) {
-        if (statusTranslation.isDefinedAt(opStatus))
-          statusTranslation(opStatus) match {
-            case CASNotFoundStatus =>
-              result.tryComplete(Success(SuccessfulResult(key, None)))
-            case CASSuccessStatus =>
-            // nothing
-            case failure =>
-              result.tryComplete(Success(FailedResult(key, failure)))
-          }
-
-        else
-          throw new UnhandledStatusException(
-            s"${opStatus.getClass}(${opStatus.getMessage})"
-          )
+        handleStatus(opStatus, key, result) {
+          case CASNotFoundStatus =>
+            result.tryComplete(Success(SuccessfulResult(key, None)))
+          case CASSuccessStatus =>
```

Comment:
Here we had a `case failure` and was also the possibility we cannot translate the `opStatus` that we get. Aren't those branches necessary anymore? If they are not, that's fine.